docs/arguments.md

```diff
@@ -118,6 +118,16 @@ Environment Variable: DOCKER_API_VERSION
              Default: "1.24"
 ```
 
+## Include restarting
+Will also include created and exited containers.
```

Comment:
Ah damn, my c&p skills failed me, this should be `Will also include restarting containers`, and of course I only see it after it's published 🤦 my bad

@simskij worth another PR or do you wanna just commit this directly?

Reply:
I'll fix it. Thanks for letting me know 👍🏼